collectors/macos.plugin/multi_metadata.yaml

```
@@ -42,14 +48,208 @@ modules:
         list: []
       configuration:
         file:
-          name: ""
-          description: ""
+          name: "netdata.conf"
+          section_name: |
+            - `[plugin:macos:sysctl"]` - Enable or disable monitoring for network, swap, loadavg, and boot time.
+            - `[plugin:macos:mach_smi]` - Enable or disable monitoring for CPU and Virtual memory.
+            - `[plugin:macos:iokit]` - Enable or disable monitoring for storage device.
```

let's not add strings here, this should just be the section name, in this case we can do the following, and add info further down to indicate this has three different sections.


```suggestion
          section_name: [plugin:macos:...]
```